.github/workflows/check-rust.yml

```diff
@@ -20,6 +20,13 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
+  security_audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/audit-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}

```

Comment:
i'm not sure we should be adding our token here.  can we do this in a way that would work from a fork PR?

Reply:
@romoh what happens if the token is removed?

Reply:
The token is only required for generating a report of the vulnerabilities and warnings as an output of the action. If the token is not found an stdout error is shown but doesn't impact the result of the audit step.  I sent a test PR to confirm that this is indeed the case.

@kate-goldenring, to answer your question, We can remove the token if we don't care about the reporting part. I would keep it there just in case github figured how to get this working at some point, the reporting will light up automatically.

Reply:
@romoh thanks for checking. That sounds like a good plan to me

Reply:
you are still going to add a comment here with that information, right?